references/tags.tsv

```diff
@@ -198,6 +198,7 @@ Seide2014_parallel	doi:10.1109/ICASSP.2014.6853593
 Setty2015_seqgl	doi:10.1371/journal.pcbi.1004271
 Selvaraju2016_grad	arxiv:1610.02391
 Shaham2016_batch_effects	arxiv:1610.04181
+Shapely_wikipedia	url:https://en.wikipedia.org/wiki/Shapley_value
```

Comment:
Is the Hazewinkel reference on the Wikipedia page suitable instead of the Wikipedia page itself?